addons/sale_timesheet/views/project_update_templates.xml

```diff
@@ -31,7 +31,7 @@
 <div name="profitability" t-if="show_profitability">
 <t t-if="project.analytic_account_id and project.allow_billable and user.has_group('project.group_project_manager')" name="costs">
 <h3 style="font-weight: bolder"><u>Profitability</u></h3>
-<t t-if="project.analytic_account_id.line_ids"> The cost of the project is now at <t t-esc="profitability['costs_formatted']"/>, for a revenue of <t t-esc="profitability['revenues_formatted']"/>, leading to a
+The cost of the project is now at <t t-esc="profitability['costs_formatted']"/>, for a revenue of <t t-esc="profitability['revenues_formatted']"/>, leading to a
```

Comment:
These changes don't respect the stable policy because a custom module could do a xpath to select that `t` or adds something next to that, for instance.

The risk is really low, I agree, but I think we should keep the `t` tag in the versions and remove it in master.

Reply:
Thanks for the review. I v just rebased the branch to master.

Reply:
cc @abla001